resin_calculator/artifact_calc_test.py

```diff
@@ -12,4 +12,24 @@ def test_validate_rarity():
     assert artifact_calc.validate_rarity("3") == True
     assert artifact_calc.validate_rarity("2") == False
     assert artifact_calc.validate_rarity("1") == False
-    assert artifact_calc.validate_rarity("6") == False
\ No newline at end of file
+    assert artifact_calc.validate_rarity("6") == False
+
+def test_validate_set_selection_five():
+    valid_sets = ["gladiator's", "wanderer's troupe", "viridescent venerer", "thundering fury", "thundersoother"
+                  "crimson witch of flames", "lavawalker", "archaic petra", "retracing bolide", "maiden beloved"
+                  "noblesse oblige", "bloodstained chivalry"]
+    assert artifact_calc.validate_set_selection("gladiator's", valid_sets) == True
+    assert artifact_calc.validate_set_selection("instructor", valid_sets) == False
+    
```

Comment:
I'm not sure if this was the best way to test the differences in valid sets by artifact rarity. Would appreciate any ideas